convert2rhel/main.py

```diff
@@ -98,6 +100,11 @@ def main_locked():
 
     pre_conversion_results = None
     process_phase = ConversionPhase.POST_CLI
+
+    # since we now have root, we can add the FileLogger
+    # and also archive previous logs
+    finalize_logger("convert2rhel.log", logger_module.LOG_DIR)
```

Comment:
I'm having a bit of difficulty to wrap my head around the name here... It doesn't seem to me that it is finalizing the logger instantiated before, but rather, appending a file handler logger to it. 

Maybe I'm missing something?

Reply:
It is more or less the same as logger is a global concept for the module name you put in. I'll rename it to match what it does more accurately